db/db_impl.cc

```diff
@@ -2417,6 +2417,10 @@ Status DBImpl::SetOptions(ColumnFamilyHandle* column_family,
     s = cfd->SetOptions(options_map);
     if (s.ok()) {
       new_options = *cfd->GetLatestMutableCFOptions();
+      // Append new version to recompute compaction score.
+      VersionEdit dummy_edit;
```

Comment:
Just check. An "empty" version edit will still have prev_log_number, next_file_number and last_sequence, and encode to non-empty string. Seems to be fine?

https://github.com/facebook/rocksdb/blob/master/db/version_edit.cc#L87-L95